benchmarks/perf-tool/okpt/test/steps/steps.py

```diff
@@ -278,21 +278,23 @@ def __init__(self, step_config: StepConfig):
         self.dataset = parse_dataset(dataset_format, dataset_path,
                                      Context.INDEX)
 
+        self.doc_count = parse_int_param('doc_count', step_config.config, {},
+                                         self.dataset.size())
+        self.doc_count = min(self.doc_count, self.dataset.size())
```

Comment:
we should avoid overriding variable values. 
```suggestion
        doc_count_param_value = parse_int_param('doc_count', step_config.config, {},
                                         self.dataset.size())
        self.doc_count = min(doc_count_param_value, self.dataset.size())
```

Reply:
Will fix